coaster/sqlalchemy/mixins.py

```diff
@@ -249,18 +255,19 @@ def url_for(self, action='view', **kwargs):
         return url_for(endpoint, **params)
 
     @classmethod
-    def is_url_for(cls, _action, _endpoint=None, _external=None, **paramattrs):
+    def is_url_for(cls, _action, _endpoint=None, _app=None, _external=None, **paramattrs):
         """
         View decorator that registers the view as a :meth:`url_for` target.
         """
         def decorator(f):
             if 'url_for_endpoints' not in cls.__dict__:
```

Comment:
Is this possible anymore? as you're defining `url_for_endpoints` above in the class directly now?

Reply:
We're checking `cls.__dict__`, which is not inherited. We cannot mutate the parent class's attribute. The only way this will already be present is if the subclass directly specifies it.

Reply:
Ah ok.